source/layouts/has_both_sidebars.haml

```diff
@@ -14,6 +14,7 @@
 
             .sl-l-large-holy-grail__main.docSearch-content
               - container = current_page.data.no_container ? '' : 'sl-l-container sl-l-container--small'
+              - container << ' sl-c-index' if current_page.data.intro
```

Comment:
`sl-c-` prefix means component. The `index` here is not really a component in itself.

I think I'd rather see this as a modifier on the intro itself since all it does it modify it's size.

So it would have both these classes:
so: `sl-c-introduction sl-c-introduction--large`.

And then the CSS changed to not be a nested style but a modified style.

Reply:
> `sl-c-` prefix means component. The `index` here is not really a component in itself.

Got it, that makes sense.

> I think I'd rather see this as a modifier on the intro itself since all it does it modify it's size.
> 
> So it would have both these classes:
> so: `sl-c-introduction sl-c-introduction--large`.
> 
> And then the CSS changed to not be a nested style but a modified style.

I think my choice of names may have been confusing here... the "intro" field I'm adding is different than the "introduction" field you added earlier. It's meant to serve as a flag that the entire page serves as an introduction, and so the whole thing should be bigger. I've renamed the field to "overview" to make that more clear, and changed the class to be `sl-l-container--overview` to make it clear that it's modifying the layout.